packages/web/src/press/PressPage.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+import { H1 } from 'src/fonts/Fonts'
+import OpenGraph from 'src/header/OpenGraph'
+import { I18nProps, NameSpaces, withNamespaces } from 'src/i18n'
+import { Cell, GridRow, Spans } from 'src/layout/GridRow'
+import SideTitledSection from 'src/layout/SideTitledSection'
+import { HEADER_HEIGHT } from 'src/shared/Styles'
+import { fonts, standardStyles, textStyles } from 'src/styles'
+import { HelpfullLink } from 'src/terms/HelpfullLink'
+import { Languages } from 'src/utils/languages'
+
+export interface PressArticleFields {
+  date: string
+  publication: string
+  title: string
+  link: string
+  language: string
+}
+
+interface Props {
+  press: PressArticleFields[]
+}
+
+class PressPage extends React.PureComponent<I18nProps & Props> {
+  static async getInitialProps(context) {
+    let press = []
+    try {
+      if (context.req) {
+        const getpress = await import('src/../server/fetchPress')
+        press = await getpress.default()
```

Comment:
Can you clarify what's going on here?

Reply:
added a comment. 

on server import module and call code directly. on client call the https endpoint

Reply:
import dynamically so code isn't imported on browser.